### PR TITLE
[Bug][Connector-v2][PulsarSource]Missing SLF4JBridgeHandler class causes task start failure

### PR DESCRIPTION
**Pull request by lightzhao**
## Purpose of this pull request
Problem：PulsarSource failed to start under the flink engine. The org/slf4j/bridge/SLF4JBridgeHandler class is missing.
image
Resolution: not excluded org.slf4j:jul-to-slf4j in pom.xml

## Check list

* [ ] Code changed are covered with tests, or it does not need tests for reason:
* [ ] If any new Jar binary package adding in your PR, please add License Notice according
  [New License Guide](https://github.com/apache/incubator-seatunnel/blob/dev/docs/en/contribution/new-license.md)
* [ ] If necessary, please update the documentation to describe the new feature. https://github.com/apache/incubator-seatunnel/tree/dev/docs
* [ ] If you are contributing the connector code, please check that the following files are updated:
  1. Update change log that in connector document. For more details you can refer to [connector-v2](https://github.com/apache/incubator-seatunnel/tree/dev/docs/en/connector-v2)
  2. Update [plugin-mapping.properties](https://github.com/apache/incubator-seatunnel/blob/dev/plugin-mapping.properties) and add new connector information in it
  3. Update the pom file of [seatunnel-dist](https://github.com/apache/incubator-seatunnel/blob/dev/seatunnel-dist/pom.xml)
* [ ] Update the [`release-note`](https://github.com/apache/incubator-seatunnel/blob/dev/release-note.md).

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -585,7 +585,15 @@
...
-                                <exclude>org.slf4j:*</exclude>
+                                <!-- not excluded: jcl-over-slf4j(commons-logging to slf4j bridge) -->
+                                <exclude>org.slf4j:slf4j-api</exclude>
+                                <exclude>org.slf4j:slf4j-jdk14</exclude>
+                                <exclude>org.slf4j:slf4j-jcl</exclude>
+                                <exclude>org.slf4j:slf4j-nop</exclude>
+                                <exclude>org.slf4j:slf4j-simple</exclude>
+                                <exclude>org.slf4j:slf4j-reload4j</exclude>
+                                <exclude>org.slf4j:slf4j-log4j12</exclude>
+                                <exclude>org.slf4j:log4j-over-slf4j</exclude>
```
Comment (hailin0) [inline pom.xml:596]: revert

Changing here has no effect

**Review by hailin0 — COMMENTED** [review]
1. enable this dependency
https://github.com/apache/incubator-seatunnel/blob/dev/pom.xml#L185

2. add dependency to this location
https://github.com/apache/incubator-seatunnel/blob/dev/pom.xml#L487
```xml
            <dependency>
                <groupId>org.slf4j</groupId>
                <artifactId>jul-to-slf4j</artifactId>
            </dependency>
```

3. add pulsar connector e2e
https://github.com/apache/incubator-seatunnel/tree/dev/seatunnel-e2e/seatunnel-connector-v2-e2e

File: pom.xml (hunk 1 above)
Comment (lightzhao) [reply pom.xml:596]: Testing with your method will still be excluded, as shown in the screenshot below.
image
My method is effective, as shown in the screenshot below.
image

Comment (hailin0) [reply pom.xml:596]: Connector run on multiple platforms, so should not contain any logging framework

Comment (lightzhao) [reply pom.xml:596]: > 

ok, Is this problem to be handled by users? or are there other solutions?

Comment (hailin0) [reply pom.xml:596]: > 1. enable this dependency
>    https://github.com/apache/incubator-seatunnel/blob/dev/pom.xml#L185
> 2. add dependency to this location
>    https://github.com/apache/incubator-seatunnel/blob/dev/pom.xml#L487
> 
> ```
>             <dependency>
>                 <groupId>org.slf4j</groupId>
>                 <artifactId>jul-to-slf4j</artifactId>
>             </dependency>
> ```
> 
> 3. add pulsar connector e2e
>    https://github.com/apache/incubator-seatunnel/tree/dev/seatunnel-e2e/seatunnel-connector-v2-e2e

Here are my test results

```shell
./mvnw -T 1C clean package -pl seatunnel-core/seatunnel-flink-starter/seatunnel-flink-13-starter -am -Dmaven.test.skip=true -Pseatunnel
```

image

Comment (lightzhao) [reply pom.xml:596]: ok，thanks.

**Pull request closed without merging**